packages/ds-healthcare-gov/src/styles/settings/_healthcare-defaultTheme.scss

```diff
@@ -0,0 +1,20 @@
+$color-primary: #046791;
```

Comment:
This file should be removed, was created long ago and was likely just hanging out in your filesystem :)